Paltineanu Rares-Mihai/3.16 - curs4/main.c

```diff
@@ -0,0 +1,110 @@
+/**Given two strings of n and m integer elements,  compute:
+a) The string that contains all the elements belonging to both strings.
+b)The string of all the elements of the two given strings, written once.
+c) The string of the elements from the first string, without the elements that are also in the second string.
+*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void read(char s1[200], char s2[200], int n, int m)
```

Comment:
The usual syntax when dealing with arrays of any kind is `char * s1` instead of `char s1[200]`
Rarely will you use static length size for the arrays, but I'll tell you more in future labs